File: Java/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/manzanoPag25e26/ExercicioF.java
package manzanoPag25e26;

import java.util.Scanner;

/*f) Ler dois valores (inteiros, reais ou caracteres) para as variáveis A e B, e efetuar a troca dos valores de
forma que a variável A passe a possuir o valor da variável B e a variável B passe a possuir o valor da
variável A. Apresentar os valores trocados */

public class ExercicioF {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		int valorA, valorB, trocaValoresA, trocaValoresB;

		System.out.println("Digita o valor de A:");
		valorA = sc.nextInt();
		System.out.println("Digita o valor de B:");
		valorB = sc.nextInt();

		trocaValoresA = valorA;
		trocaValoresB = valorB;

		valorA = trocaValoresB;
		valorB = trocaValoresA;

		System.out.println("O valor de A é " + valorA);
		System.out.println("O valor de B é " + valorB);

		sc.close();

	}

}

File: ExerciciosJavaScript(LógicadeProgramacao)/exercicios pag 25 e 26(Manzano)/js/exercicioF.js
/*f) Ler dois valores (inteiros, reais ou caracteres) para as variáveis A e B, e efetuar a troca dos valores de
forma que a variável A passe a possuir o valor da variável B e a variável B passe a possuir o valor da
variável A. Apresentar os valores trocados */

var valorA = parseInt(prompt("Digita o valor de A:"))
var valorB = parseInt(prompt("Digita o valor de B:"))

var trocaValoresA = valorA 
var trocaValoresB = valorB  

valorA = trocaValoresB
valorB = trocaValoresA

alert ("O valor de A é "+valorA)
alert ("O valor de B é "+valorB)